backend/users/migrations/0003_user_username.py

```diff
@@ -15,6 +15,10 @@ class Migration(migrations.Migration):
         migrations.AddField(
             model_name='user',
             name='username',
-            field=models.CharField(blank=True, max_length=100, null=True, verbose_name='username'),
+            field=models.CharField(
```

Comment:
please revert this one, change a migration already applied isn't a good idea 😆 (we must exclude migrations from flake8/pep8/whatever)